xwalk_android.gypi

```diff
@@ -213,6 +213,58 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+        'stamp': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/gen.stamp',
+      },
+      'actions': [
+        {
```

Comment:
In addition to this action, you should probably look at `src/build/rmdir_and_stamp.py` and how it is used, for example, in `src/third_party/mojo/mojom_bindings_generator.gypi`. The idea is that you wipe out `assets_dir` before working on the files you copy to it, otherwise you risk having stale files there in the future that can break incremental builds or generate wrong APKs.